### PR TITLE
Add playlist ID's to channels.JSON & setup createPages in gatsby-node

### PR DESCRIPTION
**Pull request by Brandon-Gottshall**

### CODE REVIEW

**Comment by Brandon-Gottshall** [comment]
Finished updates. Failing Checks.

File: src/templates/video-template.js [hunk 1]
```diff
@@ -40,18 +45,18 @@ const VideoTemplate = ({ pageContext: { channelTitle, videoId } }) => {
...
         </p>
       </div>
-      <div className="w-screen hidden lg:flex">
+      <div className="hidden w-screen lg:flex">
         <div id="video-player" className="relative flex-grow flex-shrink h-screen">
           <div className="absolute w-full h-24 bg-black">
             {isMuted && (
               <button
-                className="bg-white rounded-md w-48 h-12 m-5 flex items-center justify-evenly font-bold"
+                className="flex items-center w-48 h-12 m-5 font-bold bg-white rounded-md justify-evenly"
```
Comment (jmg8766) [inline src/templates/video-template.js:59]: Is your formatter shifting these class names around? doesn't look like this line changed except for the order of it's classes.
I wouldn't mind having a prettier plugin setup to sort classes in alphabetical order, but that should be done in a separate PR, In this situation it just makes it harder to narrow down what has changed

File: src/templates/video-template.js [hunk 2]
```diff
@@ -10,11 +10,16 @@ import PlayerControls from "../components/PlayerControls"
 import { sample, shuffle } from "lodash"
 import "tailwindcss/tailwind.css"
 
-const VideoTemplate = ({ pageContext: { channelTitle, videoId } }) => {
-  const activeChannel = channels.find(({ title }) => title === channelTitle)
+const VideoTemplate = ({ pageContext }) => {
+  console.log(pageContext)
```
Comment (jmg8766) [inline src/templates/video-template.js:14]: remove console.log

File: gatsby-node.js [hunk 3]
```diff
@@ -1,20 +1,32 @@
...
-        context: { channelTitle: channel.title, videoId },
+  // call createPage for every single video with info about videoId and title
+  const channels = await require("./data/channels.json").filter(({ playlistId }) => playlistId)
+  await Promise.all(channels.map(async ({ playlistId }) => {
+      const playlistData = await (
+        await axios.get(
+          `https://www.googleapis.com/youtube/v3/playlists?part=snippet&id=${playlistId}&key=${process.env.REACT_APP_YOUTUBE_API_KEY}`
+        )
+      ).data
+      // console.log(playlistData.items[0].snippet)
```
Comment (jmg8766) [inline gatsby-node.js:12]: remove this line

File: data/channels.json [hunk 4]
```diff
@@ -1,426 +1,185 @@
 [
     {
         "title": "History of Marketing",
-        "description": "Deep dives into the marketing stories of yesterday",
+        "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
```
Comment (jmg8766) [inline data/channels.json:4]: why did you replace description with stock text?

File: data/channels.json [hunk 5]
```diff
@@ -1,426 +1,185 @@
...
-            "tgoUa2wWvAQ",
-            "-VUXipOiX2U",
-            "PFbCGT_AWBI",
-            "HSRieuzms24",
-            "uyCl3BdlICY",
-            "Rxm13DpWGf8",
-            "eTcV-7Ubzpg",
-            "480rCl8h2Sw"
-        ]
+            "V9Ob0tjdr5M"
```
Comment (jmg8766) [inline data/channels.json:26]: Why did you remove some of the videoIds? I'd expect we either leave them in just in case we need them again or remove them completely

File: src/templates/video-template.js (hunk 1 above)
Comment (Brandon-Gottshall) [reply src/templates/video-template.js:59]: I use [headwind](https://marketplace.visualstudio.com/items?itemName=heybourn.headwind) on all my projects. It is still enabled and that's what's been sorting your classes. Apologies if it's confusing, it enforces consistently and gets rid of the headache of manually alphabetical sorting especially in regards to media queries.

File: data/channels.json (hunk 4 above)
Comment (Brandon-Gottshall) [reply data/channels.json:4]: This was unintentional. Will revert.

File: data/channels.json (hunk 5 above)
Comment (Brandon-Gottshall) [reply data/channels.json:26]: The video ID's were updated via the YouTube API to directly reflect the current state of the playlists. If a videoID was removed, it's because it's no longer in the playlist. In the future I believe these should be somehow cut out completely as they're essentially hard coded and newly added videos will not be randomly navigated to upon page load.

File: src/templates/video-template.js (hunk 2 above)
Comment (Brandon-Gottshall) [reply src/templates/video-template.js]: Removed.

File: gatsby-node.js (hunk 3 above)
Comment (Brandon-Gottshall) [reply gatsby-node.js]: Removed.
